UnityProject/Assets/Scripts/Items/Headset.cs

```diff
@@ -4,15 +4,39 @@
 /// <summary>
 ///     Headset properties
 /// </summary>
-public class Headset : NetworkBehaviour
+public class Headset : NetworkBehaviour, ICheckedInteractable<HandApply>
```

Comment:
Think you want HandActivate?

Reply:
Well hand apply is using it on something, hand activate is clicking on it in hand.